src/mmw/js/src/modeling/models.js

```diff
@@ -494,6 +494,12 @@ var ScenarioModel = Backbone.Model.extend({
             ]
         });
 
+        // If the census of the Area of Interest is available, then
+        // attach it so that it does not need to be recomputed.
+        if (attrs.aoi_census) {
+            this.set('aoi_census', attrs.aoi_census);
+        }
```

Comment:
Does the above code block (488 - 495) not do the job of setting `aoi_census` as well as any other attributes passed in on initialization?